docs/resources/ddm_instance.md

```diff
@@ -0,0 +1,174 @@
+
+* `param_group_id` - (Optional, String, ForceNew) Specifies the ID of parameter group.
+
+  Changing this parameter will create a new resource.
+
+* `time_zone` - (Optional, String, ForceNew) Specifies the time zone.
+
+  Changing this parameter will create a new resource.
+
+* `admin_user_name` - (Optional, String, ForceNew) Specifies the username of the administrator.
+  The user name starts with a letter, consists of 1 to 32 characters, and can contain only letters,
+  digits, and underscores (_).
+
+  Changing this parameter will create a new resource.
+
+* `admin_user_password` - (Optional, String) Specifies the password of the administrator.
+  The password consists of 8 to 32 characters, and must be a combination of uppercase letters,
+  lowercase letters, digits, and the following special characters: ~!@#%^*-_=+?.
+
+* `charging_mode` - (Optional, String, ForceNew) Specifies the charging mode of the DDM instance.
+  Valid values are **prePaid** and **postPaid**, defaults to **postPaid**.
+
+  Changing this parameter will create a new resource.
+
+* `period_unit` - (Optional, String, ForceNew) Specifies the charging period unit.
+  Valid values are **month** and **year**. This parameter is mandatory if `charging_mode` is set to **prePaid**.
+
+  Changing this parameter will create a new resource.
+
+* `period` - (Optional, Int, ForceNew) Specifies the charging period.
+  If `period_unit` is set to **month**, the value ranges from 1 to 9.
+  If `period_unit` is set to **year**, the value ranges from 1 to 3.
+  This parameter is mandatory if `charging_mode` is set to **prePaid**.
+
+  Changing this parameter will create a new resource.
+
+* `auto_renew` - (Optional, String, ForceNew) Specifies whether auto renew is enabled.
+  Valid values are **true** and **false**. Defaults to **false**.
+
+  Changing this parameter will create a new resource.
+
+* `delete_rds_data` - (Optional, String) Specifies whether data stored on the associated DB instances is deleted.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `status` - Indicates the status of the DDM instance.
+
+* `access_ip` - Indicates the address for accessing the DDM instance.
+
+* `access_port` - Indicates the port for accessing the DDM instance.
+
+* `engine_version` - Indicates the engine version.
+
+* `nodes` - Indicates the node information.
+  The [NodeInfoRef](#DdmInstance_NodeInfoRef) structure is documented below.
+
+<a name="DdmInstance_NodeInfoRef"></a>
+The `NodeInfoRef` block supports:
+
+* `status` - Indicates the status of the DDM instance node.
+
+* `port` - Indicates the port of the DDM instance node.
+
+* `ip` - Indicates the IP address of the DDM instance node.
+
+## Timeouts
+
+This resource provides the following timeouts configuration options:
+
+* `create` - Default is 15 minutes.
+* `update` - Default is 60 minutes.
+* `delete` - Default is 10 minutes.
```

Comment:
Do `update` operation need 60 minutes, If it is too long?

Reply:
if the flavor or node num is modified, then it will be accompanied by data migration, which will take more time